### ISSUE TITLE
ApiCall

### ISSUE THREAD

**Issue by arnaudskovich** [brightamouzou/infinite-scroll-react-js#1]
is ApiCall a component or a custom hook ? (I'm asking because you placed it in Components folder).

**Comment by brightamouzou** [comment]
Yes It is a component since I used useEffect. According to good manners I
would have to create a new directory for API calls I didn't create it given
the smallness of the project

Thanks for the remark.
